File: JavaScript Solutions/1046.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

let horaInicial = parseInt(lines.shift()),
    horaFinal = parseInt(lines.shift()),
    diferenca = horaFinal - horaInicial;
    
if (diferenca <= 0) {
    diferenca += 24;
}
    
console.log("O JOGO DUROU "+diferenca+" HORA(S)");

File: Python Solutions/1046.py
hora= [int(x) for x in input().split()]

if hora[0] == hora[1]:
    print("O JOGO DUROU 24 HORA(S)")

else:
    duracao= (hora[1]-hora[0]) % 24
    print(f"O JOGO DUROU {duracao} HORA(S)")
